SPECS-EXTENDED/GeoIP-GeoLite-data/GeoIP-GeoLite-data.spec

```diff
@@ -0,0 +1,314 @@
+Summary:        Free GeoLite IP geolocation country database
+Name:           GeoIP-GeoLite-data
+# The geolite databases were traditionally updated on the first Tuesday of each month,
+# hence we use a versioning scheme of YYYY.MM for the Fedora package.
+#
+# No further releases of IPv4 GeoLite Legacy databases will be made from April 2018.
+Version:        2018.06
+Release:        11%{?dist}
+# License specified at http://dev.maxmind.com/geoip/legacy/geolite/#License
+License:        CC-BY-SA
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://dev.maxmind.com/geoip/legacy/geolite/
+Source0:        %{_mariner_sources_url}/GeoIP.dat.gz
+Source1:        %{_mariner_sources_url}/GeoIPv6.dat.gz
+Source2:        %{_mariner_sources_url}/GeoLiteCity.dat.gz
+Source3:        %{_mariner_sources_url}/GeoLiteCityv6.dat.gz
+Source4:        %{_mariner_sources_url}/GeoIPASNum.dat.gz
+Source5:        %{_mariner_sources_url}/GeoIPASNumv6.dat.gz
```

Comment:
Where are these source coming from? What are the update steps for a new version of the package?